UI/window-basic-main.cpp

```diff
@@ -4282,24 +4283,16 @@ static inline enum obs_scale_type GetScaleType(ConfigFile &basicConfig)
 
 static inline enum video_format GetVideoFormatFromName(const char *name)
 {
-	if (astrcmpi(name, "I420") == 0)
+	if (astrcmpi(name, "4:2:0 8 bits (I420)") == 0)
```

Comment:
There must be a better way to do this. You can probably have a separate internal and displayed string in Qt.

Reply:
You can, but it requires either abusing translations or using a data model. The former is hacky, and the latter cannot be done from within a .ui file.

Reply:
Doesn't this also break compatibility with existing config files? We had a similar issue with making the Color Space entries more descriptive, so we use translations.

Reply:
Yes, in the previous version, existing config files will break (and i made a mistake in the previous version (changing the text order without updating this func). In the last version, only "internal" name is used.

Reply:
You won't need as many code changes if you use the translation system.

Reply:
> You won't need as many code changes if you use the translation system.

Thanks for pointing this. Much cleaner way to change text, and enable translation